drivers/gles3/rasterizer_canvas_gles3.h

```diff
@@ -219,7 +219,7 @@ class RasterizerCanvasGLES3 : public RendererCanvasRender {
 				};
 				float dst_rect[4];
 				float src_rect[4];
-				float pad[2];
+				uint32_t instance_uniforms_ofs;
```

Comment:
Not sure if I am setting this in the right place...

Reply:
No, this isn't the right place. But it makes me realize that more changes are needed than I described before, sorry. 

We need to move `specular_shininess` into a regular uniform. That will be easy because it is something that is the same for the entire draw call. Then `instance_uniforms_ofs` should go in the spot where `specular_shininess` used to be